docker-compose.yml

```diff
@@ -47,26 +34,10 @@ services:
       retries: 5
     networks:
       - app_network
-
-  pgadmin:
-    image: dpage/pgadmin4
-    ports:
-      - "5050:80"
-    environment:
-      - PGADMIN_DEFAULT_EMAIL=${PGADMIN_EMAIL}
-      - PGADMIN_DEFAULT_PASSWORD=${PGADMIN_PASSWORD}
-    volumes:
-      - pgadmin_data:/var/lib/pgadmin
-    depends_on:
-      - db
-    networks:
-      - app_network
+    restart: unless-stopped
 
 volumes:
   postgres_data:
-  pgadmin_data:
-  node_modules:
-  next-cache:
 
 networks:
   app_network:
```

Comment:
**suggestion:** We could remove the custom network and let compose to use the default project one.

```suggestion
```

We can remove this section in both services:
```
    networks:
      - app_network
```